configure.ac

```diff
@@ -603,7 +603,7 @@ AC_ARG_ENABLE(waf_json_logging,
 [
```

Comment:
Could you split another commit for BUG FIX?

Reply:
Sure